src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.model;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package statement to adhere to the common conventions.